core/lib/tee_verifier/src/lib.rs

```diff
@@ -191,36 +216,43 @@ fn map_log_tree(
     idx: &mut u64,
 ) -> anyhow::Result<TreeInstruction> {
     let key = storage_log.key.hashed_key_u256();
-    Ok(match (storage_log.is_write(), *tree_log_entry) {
-        (true, TreeLogEntry::Updated { leaf_index, .. }) => {
+    let tree_instruction = match (storage_log.is_write(), storage_log.kind, *tree_log_entry) {
```

Comment:
AFAIU, `storage_log.kind` is not matched now, so it can be removed from the matched expression.

Reply:
removed